2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,25 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let myFavoriteRecipe = {
+  title:"Cheesecake",
+  servings: 6,
+  ingredients: [
+    "egg", 
+    "icing sugar",
+    "cream cheese",
+    "biscuits",
+    "strawberry sauce"
+  ]
+}
+
+console.log(myFavoriteRecipe.title);
+console.log("Serves: " + myFavoriteRecipe.servings);
+console.log("Ingredients: ");
+console.log(myFavoriteRecipe.ingredients[0]);
```

Comment:
remember to use an array method if you want to do something to every element of the array, e.g.
```
myFavoriteRecipe.ingredients.forEach(ingredient => console.log(ingredient));
```